lib/red_cross.rb

```diff
@@ -11,13 +11,21 @@
 require 'red_cross/trackers/monitor_tracker'
 require 'red_cross/trackers/application_tracker'
 
+include ProductionE2EFilter
+
 module RedCross
     class << self
       def track(attrs, topic = '')
+        if is_filtering_enabled?
```

Comment:
can be one liner
`return if is_e2e_test_flow?(attrs[:properties]) if is_filtering_enabled?`